sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/jit/JitTimeTests.java

```diff
@@ -39,10 +38,4 @@ public void timestampAddTableTest() {
                 new DBSPTimestampLiteral(600100)
         );
     }
-
-    @Test @Ignore("IString parsing not supported https://github.com/feldera/feldera/issues/338")
-    public void castTimestampToStringToTimestamp() {
```

Comment:
Why are these tests being removed?

Reply:
They were ignored. Not any longer

Reply:
But it looks like the tests are removed, not just being un-ignored?

Reply:
The inherited test was ignored. It is still inherited.